rtl/ip/system_info/data/system_info.hjson

```diff
@@ -0,0 +1,61 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+{
+  name:               "system_info",
+  human_name:         "System Information",
+  one_line_desc:      "This module contains information about the system, for example the git hash.",
+  one_paragraph_desc: "",
+  cip_id:             "1",
+
+  revisions: [
+  {
+    version:            "1.0.0",
+    life_stage:         "L2",
+    design_stage:       "D1",
+    verification_stage: "V1",
+    notes:              ""
+  }
+  ]
+
+  clocking: [
+    {clock: "clk_i", reset: "rst_ni", primary: true},
+  ]
+  bus_interfaces: [
+    { protocol: "tlul", direction: "device" }
+  ],
+  available_input_list: [],
+  available_output_list: [],
+  interrupt_list: [],
+  regwidth: "32",
+  registers: [
+    { name: "RTL_COMMIT_HASH",
+      desc: '''This contains the git hash of the Sonata System repository when this bitstream was generated.
+               This is useful to determine the configuration of the RTL.''',
+      swaccess: "ro",
+      hwaccess: "hwo",
+      hwext: "true",
```

Comment:
May want to use this with `RTL_COMMIT_DIRTY` too to remove the `de` bit from `system_info_hw2reg_rtl_commit_dirty_reg_t`

Reply:
I think not setting the `de` bit is actually causing `system_info_check` to always report "clean". I can't get it to report "DIRTY!" even when I see it trying to set that bit in the generated system_info.sv file.

Reply:
Good point. I had tested the commit part quite thoroughly but not the dirty bit. I also took the liberty to make the dirty bit reset to 1, not 0.